src/databases/models/UserTopics.js

```diff
@@ -16,25 +16,27 @@ module.exports = (sequelize, DataTypes) => {
       user_topics_id: {
         type: DataTypes.STRING,
         allowNull: false,
-        primaryKey: true,
+        primaryKey: true
       },
-      user_id: { type: DataTypes.STRING, allowNull: false },
-      topic_id: { type: DataTypes.BIGINT, allowNull: false },
+      user_id: {type: DataTypes.STRING, allowNull: false},
+      topic_id: {type: DataTypes.BIGINT, allowNull: false},
+      notify_user: {type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false},
+      is_anonymous: {type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false},
       createdAt: {
         type: DataTypes.DATE,
-        field: "created_at",
-        allowNull: false,
+        field: 'created_at',
+        allowNull: false
       },
       updatedAt: {
         type: DataTypes.DATE,
-        field: "updated_at",
-        allowNull: false,
-      },
+        field: 'updated_at',
+        allowNull: false
+      }
     },
     {
       sequelize,
-      modelName: "UserTopic",
-      tableName: "user_topics",
+      modelName: 'UserTopic',
+      tableName: 'user_topics'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `UserTopic` model are significant. Two new fields, `notify_user` and `is_anonymous`, have been added with a default value of `false`. These fields are not nullable, which means they must always have a value when creating or updating a record in the `user_topics` table. This is a good practice as it ensures data consistency.

However, I noticed that there's no validation for the `user_id` and `topic_id` fields. It's important to validate these fields to ensure that they correspond to valid users and topics respectively. You can add a foreign key constraint to these fields to ensure this.

Here's how you can add a foreign key constraint:

```diff
      user_id: {
        type: DataTypes.STRING,
        allowNull: false,
+        references: {
+          model: 'users', // name of your users table
+          key: 'id' // field of users table that is being referred to
+        }
      },
      topic_id: {
        type: DataTypes.BIGINT,
        allowNull: false,
+        references: {
+          model: 'topics', // name of your topics table
+          key: 'id' // field of topics table that is being referred to
+        }
      },
```

Please note that the above suggestion assumes that your users table is named 'users' and your topics table is named 'topics'. Please replace these with the actual names if they are different. Also, replace 'id' with the actual primary key field name if it's different.


<!-- This is an auto-generated comment by OSS CodeRabbit -->